develop/schemadoc/Protocol/Protocol.Params.Param.Alarm-type.md

```diff
@@ -30,8 +30,9 @@ Prior to DataMiner 10.1.9, when a parameter has a (smart) baseline value specifi
 
 > [!NOTE]
 >
-> - The alarm template baseline editor will not allow you to change the monitoring type (Normal, Relative, Absolute or Rate).
-> - When a baseline is specified in a protocol, the baseline value is stored in a separate parameter. Although you should specify a read parameter (e.g. `<Alarm type="absolute:NOMINAL_VALUE_PID,FACTOR_PID">`), make sure that read parameter has an associated write parameter. Otherwise, it will not be possible to update the baseline value stored in that parameter. Also, the parameter in which the baseline value is stored must not have any restrictions (e.g. step size, number of decimals, high/low range, etc.).
+> - The alarm template baseline editor will not allow you to change the monitoring type (Normal, Relative, Absolute, or Rate).
+> - When a baseline is specified in a protocol, the baseline value is stored in a separate parameter. Although you should specify a read parameter (e.g. `<Alarm type="absolute:NOMINAL_VALUE_PID,FACTOR_PID">`), make sure that read parameter has an associated write parameter. Otherwise, it will not be possible to update the baseline value stored in that parameter. Also, the parameter in which the baseline value is stored must not have any restrictions (e.g. step size, number of decimals, high/low range, the interprete->rawtype tag should not be an "unsigned" number, etc.).
```

Comment:
```suggestion
> - When a baseline is specified in a protocol, the baseline value is stored in a separate parameter. Although you should specify a read parameter (e.g. `<Alarm type="absolute:NOMINAL_VALUE_PID,FACTOR_PID">`), make sure that read parameter has an associated write parameter. Otherwise, it will not be possible to update the baseline value stored in that parameter. Also, the parameter in which the baseline value is stored must not have any restrictions (e.g. step size, number of decimals, high/low range, etc.), and the Interprete.RawType tag should not be set to [unsigned number](xref:Protocol.Params.Param.Interprete.RawType#unsigned-number).
```